trainer.py

```diff
@@ -7,18 +7,29 @@
 from torch import nn, optim
 from torch.utils.tensorboard import SummaryWriter
 from tqdm.notebook import tqdm
+from transformers import ViTMAEForPreTraining
 from data import get_train_loader, get_val_loader
+from diffusion import Diffusion
+from unet import UNet
 
 class Trainer:
     def __init__(self, args):
         self.args = args
         self.device = args.device
         self.train_loader = get_train_dataloader(args)
```

Comment:
Why are the data imports and the function called here different?

Reply:
Because you wrote it like that :)

Reply:
Will add train, val, and test data loader methods to the data module in the future. This is fine for now.